cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -30,6 +29,16 @@
 evm_log = '/var/www/miq/vmdb/log/evm.log'
 
 
+@pytest.fixture
+def network_share(utility_vm):
+    utility_vm_hostname, root_password = utility_vm
+    info = {
+        'hostname': utility_vm_hostname,
+    }
+    info.update(cfme_data.utility_vm.network_share)
+    return info
```

Comment:
Thoughts on this fixture being baked into the `utility` VM fixture?

Reply:
I am not 100% sure of how the followups deps look like but I think I could do that in this PR.

Reply:
Well, it will mean to change the utility_vm fixture return values. That means modification of the restore tests. And this will likely collide with the follow-up. I would like to fix it in the followup then.

Reply:
Done in the follow-up.
https://github.com/ManageIQ/integration_tests/pull/9646/commits/32b3b811ebbe4805fe4a1cf6a4e39b15421d1c29